eventsourcing/src/main/java/org/axonframework/eventsourcing/SnapshotterSpanFactory.java

```diff
@@ -0,0 +1,31 @@
+package org.axonframework.eventsourcing;
+
+import org.axonframework.tracing.Span;
+
+/**
+ * Span factory that creates spans for the {@link Snapshotter}.
+ * You can customize the spans of the snapshotter by creating your own implementation.
+ *
+ * @author Mitchell Herrijgers
+ * @since 4.9.0
+ */
+public interface SnapshotterSpanFactory {
+    /**
+     * Creates a new {@link Span} which represents the scheduling of snapshot creation to the snapshotter's executor.
+     *
+     * @param aggregateType       The aggregate's type.
+     * @param aggregateIdentifier The aggregate's identifier.
+     * @return A {@link Span} representing the scheduling of snapshot creation.
+     */
+    Span createScheduleSnapshotSpan(String aggregateType, String aggregateIdentifier);
+
+    /**
+     * Creates a new {@link Span} which represents the actual creation of a snapshot. The creation
```

Comment:
The word "which" typically comes after a comma. If the sentence doesn't need to be broken up with a comma, you should use "that" instead:
```suggestion
     * Creates a new {@link Span} that represents the actual creation of a snapshot. The creation
```